.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16.x
+    - uses: jetli/wasm-pack-action@v0.3.0
+    - name: npm install and build
+      run: |
+        npm install
+        npm run build:v2-compat
+        npm run build:firefox
+        npm run build:chrome
+        npm run build:webext
```

Comment:
Add a step for producing an artifact here, the build will be in `dist` and all the release bundles (zips) should have the top level be the contents of the `dist` folder.

Reply:
Other than that, I think it's all good